sdk/keyvault/keyvault-keys/test/public/localCryptography.spec.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  localSupportedAlgorithms,
+  LocalSupportedAlgorithmName
+} from "../../src/localCryptography/algorithms";
+import { KeyClient, CryptographyClient, SignatureAlgorithm } from "../../src";
+import * as chai from "chai";
+import { isNode } from "@azure/core-http";
+import { createHash } from "crypto";
+import { authenticate } from "../utils/testAuthentication";
+import TestClient from "../utils/testClient";
+import { Recorder, env } from "@azure/test-utils-recorder";
+import { ClientSecretCredential } from "@azure/identity";
+const { assert } = chai;
+
+describe("Local cryptography public tests", () => {
+  const keyPrefix = `localCrypto${env.KEY_NAME || "KeyName"}`;
+  let client: KeyClient;
+  let testClient: TestClient;
+  let recorder: Recorder;
+  let credential: ClientSecretCredential;
+  let keySuffix: string;
+
+  if (!isNode) {
+    // Local cryptography is only supported in NodeJS
```

Comment:
Not blocking of this PR, but are we considering local cryptography for public key operations in the browser, or is the algorithm support just not there (even if partial)?